### DISCUSSION TITLE
Is the library still being maintained?

### DISCUSSION THREAD

**Discussion by harshitmahapatra** [facebookresearch/playtorch#216] [General]
The last commit to main was on Feb 7 2023.
There also have been no new pull requests since Feb.
The library does not yet support react native 0.71 and we are pretty close to the release of react native 0.72.

**Comment by raedle** [comment]
PlayTorch is no longer maintained, and the repository will be archived

**Reply by shahidjabbar** [reply]
Thanks! Any particular reason why it should be archived? What can we do to take it over and maintain it?

**Reply by raedle** [reply]
Archived means that the repository will be frozen in time. People can still fork the repo and continue working on it. If there is enough interest from people to keep it alive (3+ people), I can start a fork, invite interested people as maintainers, and advice on development
